Makefile

```diff
@@ -26,6 +26,9 @@ firectl.sha256:
 firectl: $(SRCFILES)
 	go build
 
+docker:
```

Comment:
Should this be called `build-in-docker` or something a little more descriptive? If I just saw the target name, I might think this was building a container image or something.

Reply:
Sure.